src/integration-test/helpers/claimStoreHelper.ts

```diff
@@ -16,8 +15,9 @@ class ClaimStoreHelper extends codecept_helper {
     const claim: Claim = await ClaimStoreClient.retrieveByReferenceNumber(claimRef, { bearerToken: claimantBearerToken })
     const pinResponse = await IdamClient.getPin(claim.letterHolderId)
     const upliftToken = await IdamClient.authenticatePinUser(pinResponse.body)
-    const defendantBearerToken = await IdamClient.upliftUser(defendantEmail, upliftToken)
-    await ClaimStoreClient.linkDefendant({ bearerToken: defendantBearerToken }, claim.externalId)
+    await IdamClient.upliftUser(defendantEmail, upliftToken)
```

Comment:
That does not look right. It looks like you start uplift process without completing it and then in next two lines you just link newly created defendant, not the one that got created in uplift process.